include/ada/url_aggregator-inl.h

```diff
@@ -1109,6 +1109,28 @@ inline std::ostream &operator<<(std::ostream &out,
                                 const ada::url_aggregator &u) {
   return out << u.to_string();
 }
+
+void url_aggregator::set_host_to_base_host(
```

Comment:
set_ prefix functions are public, and update_ ones are privates. can we change the function name and add `@private` as a comment so it won't be included in the documentation.

Reply:
I changed the name. I don't see where we use @private. The function is already private.

Reply:
Here's an example: https://github.com/ada-url/ada/blob/e685c55f1b0acfc3a661a4dad84074b0a6b04264/include/ada/url.h#L47

`@private` just hides it from doxygen.

Reply:
Yes,  but in that example, we are marking private something that is public in C++.